modules/ding_mobilesearch/ding_mobilesearch.module

```diff
@@ -538,3 +551,52 @@ function theme_ding_mobilesearch_types_table(array $variables) {
 
   return $output;
 }
+
+/**
+ * Implements hook_cron().
+ */
+function ding_mobilesearch_cron() {
+  $last_cron = variable_get('ms_update_oh', 0);
```

Comment:
Please prefix variables with the name of the responsible module. `ms_update_oh` is not very descriptive.